tutorials/prompt-scavenger.md

```
@@ -397,3 +399,80 @@ If you're up for a challenge, you can refer to the Node API [client guide](./gol
 
 - Subscribing to new prompts inside the `ce1e5714` namespace, submitting each one to GPT-3.5
 - Posting the responses back to Celestia under a different namespace.
+
+```go
+// promptGPT sends the retrieved blob data to ChatGPT and returns its response
+func promptGPT(ctx context.Context, apiKey string, prompt string) (string, error) {
+	client := openai.NewClient(apiKey)
```

Markdownlint (0.37.0)

Column: 1
Hard tabs

(MD010, no-hard-tabs)